lib/features/quiz/model/quiz.dart

```diff
@@ -0,0 +1,52 @@
+class Question {
+  final String questionText;
+  final List<Answer> answerList;
+  Question({required this.questionText, required this.answerList});
+}
+
+class Answer {
+  final String answerText;
+  final bool isCorrect;
+  Answer({required this.answerText, required this.isCorrect});
+}
+
+List<Question> getQuestions() {
+  List<Question> list = [];
+  // Adding some data
+  list.add(Question(
```

Comment:
why we need add them, can't we just create like list = [Question, Question];

Reply:
yes sure i will change it